webpack.config.js

```diff
@@ -8,14 +8,25 @@ const config = {
     path: path.resolve(__dirname, 'dist')
   },
 	module: {
-		rules: [{ test: /\.js$/, loader: 'babel-loader', exclude: /node_modules/ }]
+		rules: [
+      {
+        test: /\.js$/,
+        loader: 'babel-loader',
+        exclude: /node_modules/
+      },
+      {
+        test: /\.wasm$/i,
+        type: 'javascript/auto',
+        loader: 'file-loader',
+        options: {
+          name: '[name].[ext]'
+        }
+      }
+    ]
   },
   resolve: {
 		extensions: ['.js']
```

Comment:
Do you need to add `.wasm` into `extensions`?

Reply:
Please refer to https://webpack.js.org/configuration/resolve/#resolveextensions

Reply:
Yes we need this to resolve the [import](https://github.com/intel/webml-polyfill/blob/693d099a958f33105faa1d0422414f62f66ac67e/src/nn/tfjs/TfjsModel.js#L6) from wasm file. The tfjs-backend-wasm init will fail without this.